inc/azure_c_logging/xlogging.h

```diff
@@ -68,39 +52,15 @@ typedef void(*LOGGER_LOG_GETLASTERROR)(const char* file, const char* func, int l
 /*no logging is useful when time and fprintf are mocked*/
 #ifdef NO_LOGGING
 #define LOG(...)
-#define LogInfo(...)
-#define LogBinary(...)
 #define LogError(...)
-#define LogLastError(...)
-#define xlogging_get_log_function() NULL
-#define xlogging_set_log_function(...)
-#define LogErrorWinHTTPWithGetLastErrorAsString(...)
-#define UNUSED(x) (void)(x)
-#elif (defined MINIMAL_LOGERROR)
-#define LOG(...)
+#define LogWarning(...)
 #define LogInfo(...)
-#define LogBinary(...)
-#define LogError(...) printf("error %s: line %d\n",__FILE__,__LINE__);
+#define LogVerbose(...)
+#define LogLastError(...)
 #define xlogging_get_log_function() NULL
 #define xlogging_set_log_function(...)
 #define LogErrorWinHTTPWithGetLastErrorAsString(...)
-#define UNUSED(x) (void)(x)
-
-#elif defined(ESP8266_RTOS)
-#define LogInfo(FORMAT, ...) do {    \
-        static const char flash_str[] ICACHE_RODATA_ATTR STORE_ATTR = FORMAT;  \
-        printf(flash_str, ##__VA_ARGS__);   \
-        printf("\n");\
-    } while((void)0,0)
-
-#define LogError LogInfo
-#define LOG(log_category, log_options, FORMAT, ...)  { \
-        static const char flash_str[] ICACHE_RODATA_ATTR STORE_ATTR = (FORMAT); \
-        printf(flash_str, ##__VA_ARGS__); \
-        printf("\r\n"); \
-}
-
-#else /* NOT ESP8266_RTOS */
+#else
```

Comment:
>else [](start = 1, length = 4)

nit: can we comment what these else pre processor lines are for? I'm having trouble matching them up (could just be the diff). At least this one and the one on line 144

Reply:
Added

---
In reply to: [425318306](https://github.com/Azure/azure-c-logging/pull/3#discussion_r425318306) [](ancestors = 425318306)